cypress/features/regression/designSystem/toastStacked.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Toast Design System component stacked story
+  I want verify stacked Design System Toast component
+
+  Background: Open Design SystemToast component stacked page
+    Given I open Design Systems default_story "Toast" component docs page
```

Comment:
The file is called `toastStacked` - but you are opening `default_story`

Reply:
@damianOfKyan you can remove this file and move this code to the `designSystem/toast.feature`. You should use `id` to locate `stacked Toast`. You should put the `ID` in `\cypress\locators\toast\locators.js`